packages/browserslist-config-terra/package.json

```diff
@@ -35,7 +35,7 @@
     "lint:js:fix": "eslint --ext .js,.jsx . --fix"
   },
   "devDependencies": {
-    "@cerner/eslint-config-terra": "^5.5.1",
+    "@cerner/eslint-config-terra": "5",
```

Comment:
Why don't we want to consume the minor version dependency updates?

Reply:
Actually I think I understand now. This is just so that the version bumps don't appear in package.json and trigger a new release, but running `npm install` will still install the latest version?

Reply:
Correct! Both these syntax are identical. This is so that these dependencies don't get updated and trigger release. Since these are devDependencies, they aren't relevant for consumers and don't need to be released. e.g. #823 where only eslint-config-terra had releasable changes but everything else had to be released to due to a simple dev dependency bump.